### PR TITLE
CMSPLT-73: Upgrade uPortal-spring to fix data import

### PR DESCRIPTION
**Pull request by bjagg**
https://issues.jasig.org/browse/CMSPLT-73

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
         <slf4j.version>1.7.11</slf4j.version>
         <spring.version>4.1.5.RELEASE</spring.version>
         <uportal-maven-plugin.version>1.0.0</uportal-maven-plugin.version>
-        <uportal-libs.version>5.0.0-M4</uportal-libs.version>
+        <uportal-libs.version>5.0.0-M5</uportal-libs.version>
```
Comment (drewwills) [inline pom.xml]: I'm not aware that there is a `5.0.0-M5` -- did someone cut one?  Do you need me to do it?  (I can.)

What is different in M5?

Comment (ChristianMurphy) [reply pom.xml]: :white_check_mark: There is, you did. https://github.com/Jasig/uPortal/releases/tag/5.0.0-M5

Comment (drewwills) [reply pom.xml]: You're totally correct -- I lost count.

Comment (drewwills) [reply pom.xml]: (But I still don't understand what the meaningful difference is.)

Comment (ChristianMurphy) [reply pom.xml]: https://github.com/Jasig/uPortal/compare/5.0.0-M4...5.0.0-M5
I strongly suspect https://github.com/Jasig/uPortal/commit/d58e52afb50d30f8d3f6508dfbd513c97b6e21e8
@bjagg may be able to provide more details.

Comment (drewwills) [reply pom.xml]: I hope @bjagg will do so, if he knows.

Comment (bjagg) [reply pom.xml]: It was the issue with NewsReader and Calender already had it set to M5. I simply set it to sync with the others (unlike NewsReader).

File: src/main/resources/context/baseContext.xml [hunk 2]
```diff
@@ -107,6 +107,7 @@
 
     <bean id="jpaVendorAdapter" class="org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter" >
         <property name="generateDdl" value="false" />
+        <property name="databasePlatform" value="${hibernate.dialect}" />
```
Comment (drewwills) [inline src/main/resources/context/baseContext.xml]: Should this property _replace_ setting it somewhere else?

Comment (bjagg) [reply src/main/resources/context/baseContext.xml]: Yes, this should replace the need for hibernate.properties. Unfortunately, it does not seem to work.

**Comment by drewwills** [comment]
This change now works for me on MS SQL Server;  I also tested it on HSQLDB.

I think it's ready -- any further thoughts?

File: src/main/resources/context/baseContext.xml (hunk 2 above)
Comment (bjagg) [reply src/main/resources/context/baseContext.xml]: Replaces hibernate.properties

Comment (drewwills) [reply src/main/resources/context/baseContext.xml]: Except -- afaik the `hbm2ddl` goal (Maven) still uses hibernate.properties.

**Pull request merged**